saltproc/depcode.py

```diff
@@ -159,20 +166,346 @@ def write_mat_file(self, dep_dict, dep_end_time):
         """
 
 
+class DepcodeOpenMC(Depcode):
+    r"""Class contains information about input, output, geometry, and
+    template files for running OpenMC depletion simulations.
+    Also contains neutrons population, active, and inactive cycles.
+    Contains methods to read template and output files,
+    write new input files for OpenMC.
+
+    Attributes:
+    -----------
+    param : dict of str to type
+        Holds depletion steo parameter information
+    sim_info : dict of str to type
+        Holds simulation settings information
+    iter_inputfile : dict of str to str
+        Paths to OpenMC input files for OpenMC rerunning.
+    iter_matfile : str
+        Path to iterative, rewritable material file for OpenMC
+        rerunning. This file is modified during the simulation.
```

Comment:
Are these still used? I looks like earlier on in the file all mentions of these attributes are removed from the code.

Reply:
They are used. They used to be function parameters, but since they don't change within a simulation it makes sense to have them as attributes. Not sure where "earlier on in the file" refers to.